src/CMake/PluginMacros.cmake

```diff
@@ -85,3 +85,107 @@ MACRO(VISIT_PLUGIN_TARGET_FOLDER type pname)
     SET_TARGET_PROPERTIES(${ARGN} PROPERTIES FOLDER "plugins/${type}/${pname}")
 ENDMACRO(VISIT_PLUGIN_TARGET_FOLDER)
 
+
```

Comment:
Does this really create 6 make targets per plugin? My recollection from working in the data directory is that large numbers of targets causes CMake to take forever to complete. It already (now) can take more than a minute on my OSX system. Just curious, have you looked at performance of CMake'ing VisIt with all these new targets defined?

Reply:
to answer: yes it does create 6 targets per plot and operator plugin and 2 per database plugin